Server/mods/deathmatch/logic/luadefs/CLuaAccountDefs.cpp

```diff
@@ -173,12 +173,7 @@ int CLuaAccountDefs::GetAccountData(lua_State* luaVM)
 
     if (!argStream.HasErrors())
     {
-        auto pArgument = CStaticFunctionDefinitions::GetAccountData(pAccount, strKey);
-        if (pArgument)
-        {
-            pArgument->Push(luaVM);
-            return 1;
-        }
+        CStaticFunctionDefinitions::GetAccountData(pAccount, strKey).Push(luaVM);
     }
```

Comment:
It was returning 1 after pushing the argument. Are you sure this is correct?

Reply:
Yeah, small logic error from my side. Thanks!

Reply:
This is still missing a `return 1;` right below this push line.